components/circle/circle.app.mjs

```diff
@@ -0,0 +1,159 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "circle",
+  propDefinitions: {
+    body: {
+      type: "string",
+      label: "Body",
+      description: "The body of the post.",
+    },
+    communityId: {
+      type: "string",
+      label: "Community ID",
+      description: "Select the community you want to interact with.",
+      async options() {
+        const communities = await this.listCommunities();
+
+        return communities.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    postId: {
+      type: "string",
+      label: "Post ID",
+      description: "Select the post you want to create a comment for.",
+      async options({
+        communityId, spaceId, page,
+      }) {
+        const posts = await this.listPosts({
+          params: {
+            communityId,
+            spaceId,
+            page: page + 1,
+            status: "all",
+          },
+        });
+
+        return posts.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    spaceId: {
+      type: "string",
+      label: "Space ID",
+      description: "Select the space within the community.",
+      async options({
+        communityId, page,
+      }) {
+        const spaces = await this.listSpaces({
+          params: {
+            communityId,
+            page: page + 1,
+          },
+        });
+
+        return spaces.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    userEmail: {
+      type: "string",
+      label: "User Email",
+      description: "Email of the community member to post from. Defaults to admin if empty.",
+    },
```

Comment:
Could `userEmail` use async options by fetching community members with this endpoint?
https://api.circle.so/#a1501d36-5e30-48f3-8934-c825256f31d7